bluetoe/link_layer/include/bluetoe/peripheral_latency.hpp

```diff
@@ -26,39 +27,31 @@ namespace link_layer {
      * @brief detailed options for the peripheral latency behavior
      *
      * Every option defines a set of events / circumstances under which
-     * the link layer should listen to the very next connection event.
-     * (beside the miniumum events, the link layer has to listen to due
-     * to the periperhal latency value of the connection).
+     * the link layer should listen at the very next connection event.
+     * Regardless of the selected options, the link layer always listens
+     * for incoming PDU at each peripheral latency anchor points, which
+     * were negotiated as part of the current connection parameters.
      *
      * @note in combination, there are some options that make no sense.
      *       For example, using listen_always with some of the other
-     *       listen_* options is pointless. Also combining send_always
-     *       with send_pending_data doesn't make sense.
+     *       listen_* options is pointless.
+     *       TODO: Specify what happens if listen always is combined with
```

Comment:
@TorstenRobitzki Should we say that 'always_listen' takes over, if combined with less restrictive options?

Reply:
Yes, maybe it would even be appropriate to generate a compile time error message, if `always_listen` is combined with something else.

Reply:
I tried 2 hours to come up with a runtime check, but I did not managed to come up with a clever trick to instantiate the template, so that a static_assert would fail.